### ISSUE TITLE
Add ignoreProperties: [] to value-keyword-case

### ISSUE THREAD

**Issue by oleksiibilous** [stylelint/stylelint#2424]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

"A feature request"

> Which rule, if any, is this issue related to?

`value-keyword-case`

> What CSS is needed to reproduce this issue?

```css
.elementWithCamelCaseName {}

.element {
  composes: elementWithCamelCaseName;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "value-keyword-case": "lower",
    }
  }
}
```

> Which version of stylelint are you using?

`6.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

"WebpackPluginStylelint"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

"Yes, it's related to CSS modules"

> What did you expect to happen?

"No warnings to be flagged for composes property"

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged: Expected camelCaseName to be lowecasename"

Maybe it is a good idea to add additional option to this rule? Something like `ignoreProperties`?  I can work on PR if anyone is interested. Thank you

**Comment by evilebottnawi** [comment]
@oleksiibilous thanks for feedback, be right ignore `composes` property by default, because it is always `custom-ident`, PR :+1:

**Comment by jeddy3** [comment]
> be right ignore composes property by default,

Let's not add code to core for _methodologies_. That's what the `ignore*` options are for.

> Something like `ignoreProperties`? 

Yes, that would be perfect. @oleksiibilous Feel free [to contribute this option](https://stylelint.io/developer-guide/rules/#adding-a-option-to-an-existing-rule) via a PR.

**Comment by oleksiibilous** [comment]
`ignoreProperties` sounds good👍 Will work on it

**Comment by CAYdenberg** [comment]
@stylelint/core Should this issue be closed? It looks like it was fixed in #2937 ?
